api/db-cdr/generate-cdr/run-measurement-queries.sh

```diff
@@ -0,0 +1,412 @@
+FROM  priorstats p join overallstats o on p.stratum1_id = o.stratum1_id and p.stratum2_id = o.stratum2_id
+group by  o.stratum1_id, o.stratum2_id, o.total, o.min_value, o.max_value, o.avg_value, o.stdev_value"
+
+
+# 1900 Measurement value counts
+echo "Getting measurements value counts"
+bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
+"insert into \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.achilles_results\`
+(id, analysis_id, stratum_1, stratum_2, stratum_4, stratum_5, count_value, source_count_value)
+select 0,1900 as analysis_id,
+CAST(m1.measurement_concept_id AS STRING) as stratum_1,
+CAST(p1.gender_concept_id AS STRING) as stratum_2,
+CAST((case when cast(ar.stratum_2 as int64) > 0 then
+(case when m1.value_as_number < cast(ar.stratum_2 as int64) then cast(ar.stratum_2 as int64) else
+cast(ROUND(m1.value_as_number / cast(ar.stratum_2 as int64)) * cast(ar.stratum_2 as int64) as int64) end) else m1.value_as_number end) as STRING) as stratum_4,
+m1.unit_source_value as stratum_5,
+count(distinct p1.person_id) as count_value,
+(select COUNT(distinct m2.person_id) from \`${BQ_PROJECT}.${BQ_DATASET}.measurement\` m2 where m2.measurement_source_concept_id=m1.measurement_concept_id) as source_count_value
+from \`${BQ_PROJECT}.${BQ_DATASET}.person\` p1 inner join
+\`${BQ_PROJECT}.${BQ_DATASET}.measurement\` m1
+on p1.person_id = m1.person_id
+join \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.achilles_results\` ar on
+cast(m1.measurement_concept_id AS STRING)=ar.stratum_1
+where m1.measurement_concept_id > 0
+and m1.value_as_number is not null
+and ar.analysis_id=3000 and ar.stratum_3='Measurement'
+group by m1.measurement_concept_id,stratum_2,stratum_4,stratum_5
+union all
+select 0, 1900 as analysis_id,
+CAST(m1.measurement_source_concept_id AS STRING) as stratum_1,
+CAST(p1.gender_concept_id AS STRING) as stratum_2,
+CAST((case when cast(ar.stratum_2 as int64) > 0 then
+(case when m1.value_as_number < cast(ar.stratum_2 as int64) then cast(ar.stratum_2 as int64) else
+cast(ROUND(m1.value_as_number / cast(ar.stratum_2 as int64)) * cast(ar.stratum_2 as int64) as int64) end) else m1.value_as_number end) as STRING) as stratum_4,
+m1.unit_source_value as stratum_5,
+COUNT(distinct p1.PERSON_ID) as count_value,
+COUNT(distinct p1.PERSON_ID) as source_count_value
+from \`${BQ_PROJECT}.${BQ_DATASET}.person\` p1 inner join
+\`${BQ_PROJECT}.${BQ_DATASET}.measurement\` m1
+on p1.person_id = m1.person_id
+join \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.achilles_results\` ar on
+cast(m1.measurement_source_concept_id AS STRING)=ar.stratum_1
+where m1.measurement_source_concept_id > 0 and m1.measurement_concept_id!=m1.measurement_source_concept_id
+and m1.value_as_number is not null
+and ar.analysis_id=3000 and ar.stratum_3='Measurement'
+group by m1.measurement_source_concept_id,stratum_2,stratum_4,stratum_5"
+
+# 1900 Measurement value counts
+echo "Getting measurements value counts"
```

Comment:
What are we doing here doing same thing again ? Different group of measurements ?